docs/Advanced-Usage/FAQs.rst

```diff
@@ -53,3 +53,29 @@ interactive simulation, or you've connected the NIC to the internet), this is
 likely a bug in one of your custom bridge implementations or in FireSim. In
 fact, for a given target-design, enabling printf synthesis, assertion synthesis,
 autocounter, or Auto ILA, should not change the simulated behavior of the machine.
+
+Is there a way to compress workload results when copying back to the manager instance?
+--------------------------------------------------------------------------------------
+
+FireSim doesn't support compressing workload results before copying them back to the manager instance.
+Instead we recommend that you use a modern filesystem (like ZFS) to provide compression for you.
+For example, if you want to use ZFS to transparently compress data:
+
+#. Attach a new volume to your EC2 instance (either at runtime or during launch).
+   This is where data will be stored in a compressed format.
+#. Make sure that the volume is attached (using something like ``lsblk -f``).
+   This new volume should not have a filesystem type and should be unmounted (named something like ``nvme1n1``).
+#. Install ZFS according to <https://openzfs.github.io/openzfs-docs/Getting%20Started/RHEL-based%20distro/index.html>.
+   Check ``/etc/redhat-release`` to verify the CentOS version of the manager instance.
+#. Mount the volume and setup the ZFS filesystem with compression.
+
+.. warning::
+    Creating the zpool will destroy all pre-existing data on that partition.  Double-check that the device node is correct before running any commands.
+::
+
+    zpool create -o ashift=12 -O compression=on <POOL_NAME> /dev/nvme1n1
+    zpool list
+    zfs list
+
+#. At this point, you can use ``/<POOL_NAME>`` as a normal directory to store data into where it will
```

Comment:
Does the list numbering continue properly for the last item?  Otherwise this can be formatted as  a paragraph.

Reply:
Ok fixed. Verified locally.

![image](https://user-images.githubusercontent.com/8823803/142508125-f78542fe-d8a1-483e-905d-9387ad55e08e.png)